### PR TITLE
Update hyperlink to English terms of use

### PR DESCRIPTION
**Pull request by andrunko**
https://phabricator.endlessm.com/T29905

### CODE REVIEW

**Comment by wjt** [comment]
Please can you clarify what changes were needed here, beyond exporting the documents on (???) Fedora with a non-Flatpak'd LibreOffice?

If that's all: then don't we need to change the `generate-pdfs` target and/or `build-aux/docx-to-pdf` script, which assumes a Flatpak'd LibreOffice since dab08a4062b6ba7bf725691ebcc096bc2a9b4ba9? Otherwise when we next update these terms (perhaps to change the legal name of the LLC, which is expected to happen Any Day Now) then we are leaving a trap for an unwary editor who will use the promising-looking make target and regress this issue.

**Comment by andrunko** [comment]
> Please can you clarify what changes were needed here, beyond exporting the documents on (???) Fedora with a non-Flatpak'd LibreOffice?
> 
Yes, that was all I did, but now I found another way to make this work reliably with and without a flatpak'd libreoffice.

> If that's all: then don't we need to change the `generate-pdfs` target and/or `build-aux/docx-to-pdf` script, which assumes a Flatpak'd LibreOffice since [dab08a4](https://github.com/endlessm/eos-license-service/commit/dab08a4062b6ba7bf725691ebcc096bc2a9b4ba9)? Otherwise when we next update these terms (perhaps to change the legal name of the LLC, which is expected to happen Any Day Now) then we are leaving a trap for an unwary editor who will use the promising-looking make target and regress this issue.

I hadn't notice we had the build script to generate to pdfs, thought we were always doing it automatically. Updated the script to generate the PDFs with correct links.

File: build-aux/libreoffice/custom-profile.xcu [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<oor:items xmlns:oor="http://openoffice.org/2001/registry" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<item oor:path="/org.openoffice.Office.Common/Filter/PDF/Export"><prop oor:name="ExportLinksRelativeFsys" oor:op="fuse"><value>false</value></prop></item>
```
Comment (pwithnall) [inline build-aux/libreoffice/custom-profile.xcu:3]: This is magic. Could you add one or more comments in it explaining what it does and linking to relevant references?

Comment (andrunko) [reply build-aux/libreoffice/custom-profile.xcu:3]: hehe, indeed it is magic, truth being told I found this after lots of google searches when I finally saw a post mentioning we could disable using relative urls by using `File > Export As > Export as PDF`. Then I tried it and checked my saved profile to identify what option was set...

The closest doc I found is https://help.libreoffice.org/Common/Export_as_PDF#Export_URLs_relative_to_file_system which unfortunately doesn't mention the option.

If you still think it is worth I can add a comment and link to this page:
```
<!-- Disable option to export links relative to filesystem - https://help.libreoffice.org/Common/Export_as_PDF#Export_URLs_relative_to_file_system -->
```

Comment (pwithnall) [reply build-aux/libreoffice/custom-profile.xcu:3]: Yeah, or something like:
```
<!-- Disable option to export links relative to filesystem - https://help.libreoffice.org/Common/Export_as_PDF#Export_URLs_relative_to_file_system
     Found by using File -> Export As -> Export as PDF -> Links and checking the changes to the LibreOffice saved profile. -->
```

Comment (andrunko) [reply build-aux/libreoffice/custom-profile.xcu:3]: Done.

**Pull request merged**